src/tests.rs

```diff
@@ -84,6 +97,33 @@ However, it does nothing.",
             assert_eq!(f.mode, FileMode::from(0o120644));
         }
     });
+    
+    // Test scriptlet builder fn branches
+    let preinst = pkg.metadata.find_scriptlet(ScriptletType::PreInstall)?;
```

Comment:
Could you split these in the same way as the setters, so that the pattern is the same?